app/assets/javascripts/password_confirmation_validator.js

```diff
@@ -1,16 +1,16 @@
-$(document).on("ready page:change", function() {
-    $('[data-password-confirm]').each(function() {
+$(document).on("ready page:change paste", function () {
+    $('[data-password-confirm]').each(function () {
         var password_elm = $(this).get(0);
         var password_confirmation_elm = document.getElementById($(this).data('password-confirm'));
 
-        var validator = function() {
-            if (password_elm.value != password_confirmation_elm.value) {
+        var validator = function () {
```

Comment:
preco je tu pridana medzera? (podobne vyssie)

Reply:
RubyMine mi tak auto-formatuje, je to zle?

Reply:
nie som si teraz isty ake mame konvencie pre JS, ale ak to vsade v kode je bez medzery, tak to drzme rovnako